templates/csharp/app-mvvm/App.axaml

```diff
@@ -10,3 +11,18 @@
         <FluentTheme Mode="Light"/>
     </Application.Styles>
 </Application>
+//#else
+<Application xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:local="using:AvaloniaAppTemplate"
+             x:Class="AvaloniaAppTemplate.App"
+             RequestedThemeVariant="Light">
```

Comment:
By default, application will use system variant. If we have "Light" defined in the template, it will force light theme always.
As an alternative, we can have `="Default"` value here.

Reply:
ok, thanks didn't know that option yet. I guess it is better to set directly set (to "Default") instead of removing it because otherwise people will wonder how to set it

Reply:
Should there be a comment added to explain "Default" means follow system because it is not obvious by that name?

Reply:
Yes, makes sense